docs/collect-import-export.rst

```diff
@@ -144,14 +144,12 @@ Here are the keys for all settings and the set of values they can take:
     "general" : {
 
       // Server
-      "protocol": {"odk_default", "google_sheets"},
+      "protocol": {"odk_default"},
```

Comment:
Will we still have protocol going forward? And will it be an array.

Reply:
We will leave it for backwards-compatibility I imagine. Maybe we can remove it from docs, though.